src/JBSwapTerminal.sol

```diff
@@ -396,6 +378,10 @@ contract JBSwapTerminal is JBPermissioned, Ownable, IJBTerminal, IJBPermitTermin
     /// @param amount1Delta The amount of token 1 being used for the swap.
     /// @param data Data passed in by the swap operation.
     function uniswapV3SwapCallback(int256 amount0Delta, int256 amount1Delta, bytes calldata data) external override {
+        // Add access control: Terminal balance should be 0, this callback can only be used to sweep + callback is not
```

Comment:
is this a todo? @drgorillamd

Reply:
No, should remove the "Add", but we need to be carefull if/when refactoring